charts/sorry-cypress/templates/deployment-director.yml

```diff
@@ -17,7 +17,11 @@ spec:
       - env:
         {{- range .Values.director.ingress.hosts }}
         - name: director_URL
-          value: {{ .host }}
+          value: http://{{ .host }}
```

Comment:
@tico24, what is this variable for?

Reply:
It's a typo. It's supposed to be DASHBOARD_URL I think. Line 118: https://github.com/sorry-cypress/sorry-cypress/blob/master/kubernetes-full.yml

Reply:
It's a typo. It's supposed to be DASHBOARD_URL I think. Line 118: https://github.com/sorry-cypress/sorry-cypress/blob/master/kubernetes-full.yml

Reply:
Humm, alright! I'm removing it then.